### DISCUSSION TITLE
Deleting the gradient Tape at the end of SAC.

### DISCUSSION THREAD

**Discussion by HesNobi** [keiohta/tf2rl#133] [Q&A] [answered]
HI,
May I ask if it is necessary to delete the gradient tape at the end SAC algorithm?
Very much appreciate your helps

**Comment by keiohta** [comment, marked as answer]
I read the corresponding [API](https://www.tensorflow.org/api_docs/python/tf/GradientTape) again, and make sure we can delete the gradient tape (no need to explicitly delete it by `del tape` ).
Thanks for notifying this, and I'll reflect the changes in near future.
